php/Dockerfile

```diff
@@ -8,85 +8,85 @@ ENV TERM dumb
 RUN apt-get update && apt-get upgrade -y
 
 RUN apt-get install -y \
+    # Included applications
     git \
     mysql-client \
     php-pear \
-    tree
+    tree \
+    vim \
+    # Needed for building extensions
+    # For gd
+    libpng-dev \
+    # For imap
+    libc-client-dev \ 
+    libkrb5-dev \
+    # For intl
+    libicu-dev \
+    # For ldap
+    libldap2-dev \
+    # For mcrypt
+    libmcrypt-dev \
+    # For memcached
+    libmemcached-dev \
+    # For PDO MSSQL/SYBASE
+    freetds-common \
+    freetds-dev \
+    # For zip
+    zlib1g-dev
 
-# Install needed php extension: imap
-RUN apt-get install -y \
-    libc-client-dev \
-    libkrb5-dev && \
-    docker-php-ext-configure imap --with-kerberos --with-imap-ssl && \
+# Clean up unneccessary package information
+RUN rm -rf /var/lib/apt/lists/* && \
+    apt-get clean all
+
+# Install php extension: gd
+# Used by molbio to generate images and sparklines
+RUN docker-php-ext-install -j$(nproc) gd
+
+# Install php extension: imap
+RUN docker-php-ext-configure imap --with-kerberos --with-imap-ssl && \
```

Comment:
Could we combine all the docker-php-ext-install lines into one RUN command as well (in the future)?